### DISCUSSION TITLE
Can I use reown appkit for wallet connect?

### DISCUSSION THREAD

**Discussion by Johnson-8844** [RequestNetwork/requestNetwork#1496] [Questions] [answered]
Can I use reown appkit for wallet connect?

**Comment by aimen74** [comment, marked as answer]
@Johnson-8844 Yes, it is possible our [Payment Widget](https://github.com/RequestNetwork/web-components/tree/main/packages/payment-widget) uses WalletConnect (before it was rebranded to reown) for wallet connection.
